layouts/partials/toggle.html

```diff
@@ -0,0 +1,231 @@
+    -webkit-tap-highlight-color: transparent;
+  }
+
+  .js-toggle-screenreader-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
+
+  .js-toggle-track {
+    width: 50px;
+    height: 24px;
+    padding: 0;
+    border-radius: 30px;
+    background-color: hsl(222, 14%, 7%);
+    transition: all 0.2s ease;
+  }
+
+  .js-toggle-track-check {
+    position: absolute;
+    width: 17px;
+    height: 17px;
+    left: 5px;
+    top: 0px;
+    bottom: 0px;
+    margin-top: auto;
+    margin-bottom: auto;
+    line-height: 0;
+    opacity: 0;
+    transition: opacity 0.25s ease;
+  }
+
+  .js-toggle--checked .js-toggle-track-check {
+    opacity: 1;
+    transition: opacity 0.25s ease;
+  }
+
+  .js-toggle-track-x {
+    position: absolute;
+    width: 17px;
+    height: 17px;
+    right: 5px;
+    top: 0px;
+    bottom: 0px;
+    margin-top: auto;
+    margin-bottom: auto;
+    line-height: 0;
+    opacity: 1;
+    transition: opacity 0.25s ease;
+  }
+
+  .js-toggle--checked .js-toggle-track-x {
+    opacity: 0;
+  }
+
+  .js-toggle-thumb {
+    position: absolute;
+    top: 1px;
+    left: 1px;
+    width: 22px;
+    height: 22px;
+    border-radius: 50%;
+    background-color: #fafafa;
+    box-sizing: border-box;
+    transition: all 0.5s cubic-bezier(0.23, 1, 0.32, 1) 0ms;
+    transform: translateX(0);
+  }
+
+  .js-toggle--checked .js-toggle-thumb {
+    transform: translateX(26px);
+    border-color: #19ab27;
+  }
+
+  .js-toggle--focus .js-toggle-thumb {
+    box-shadow: 0px 0px 2px 3px rgb(255, 167, 196);
+  }
+
+  .js-toggle:active .js-toggle-thumb {
+    box-shadow: 0px 0px 5px 5px rgb(255, 167, 196);
+  }
+
+  a.nav-link.active.dark-mode{
+      color: #000000 !important;
+      /* color: #008cba !important; */
+  }
+
+  a.nav-link.dark-mode{
+      /* color: #000000 !important; */
+      color: #008cba !important;
+  }
+
+  body.dark-mode,
+  td-sidebar.dark-mode,
+  a.td-sidebar-link__section.dark-mode,
+  a.td-sidebar-link.td-sidebar-link__page.dark-mode,
+    body.dark-mode main * {
+    background: rgb(45, 45, 45);
+    color: #f5f5f5;
+  }
+  section.py-5.bg-light.dark-mode{
+      background-color: rgb(45, 45, 45) !important;
+      color: #f5f5f5;
+  }
+
+  i.fa-paperclip.dark-mode{
+      background-color: #008cba !important;
+      color: #f5f5f5;
+  }
+
+
+</style>
+
+<script>
+    var body = document.body;
+    var aref = document.getElementsByClassName('td-sidebar-link__section');
+    var are = document.getElementsByClassName('td-sidebar-link td-sidebar-link__page');
+    var anav = document.getElementsByClassName('nav-link');
+    var intro_bg = document.getElementsByClassName('py-5 bg-light');
+    var paperclip = document.getElementsByClassName('fa-paperclip');
+
+    var elems = [aref,are,anav,intro_bg,paperclip];
+	var switcher = document.getElementsByClassName('js-toggle')[0];
```

Comment:
Would be good to format them for readability purposes :+1:

Reply:
Accepted, these changes are now done